spec/customer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require './docs/customer'
+
+describe Customer do
+
+  it 'orders meals' do
+    # Arrange
+    restaurant = Restaurant.new
```

Comment:
This line makes the test for the `Customer` class depend on the `Restaurant` class working. How can you get around this?

Reply:
oh sorry I think I left that there from when I was trying to work out the next user story, I shall remove it now.